src/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
You can remove react and import naming import {StrictMode} from 'react'
```suggestion
import {StrictMode} from "react";
```